app/src/test/java/protect/card_locker/ImportExportTest.java

```diff
@@ -1017,20 +1017,20 @@ public void importV2CSV() {
         assertEquals(3, DBHelper.getGroupCount(mDatabase));
```

Comment:
Your group import code isn't dealing with the "old" V2 correctly, because after importing 0 groups exist

Reply:
This seems resolved, but only for the "old" V2... :(